core/src/main.cpp

```diff
@@ -235,24 +235,41 @@ constexpr auto default_jsonschema() -> std::string_view
                                     "minimum": 1
                                 }},
                                 "provider_url": {{
-                                    "type": "string"
+                                    "type": "string",
+                                    "format": "uri"
+                                }},
+                                "run_as_protected_resource": {{
+                                    "type": "boolean"
                                 }},
                                 "client_id": {{
                                     "type": "string"
                                 }},
-                                "redirect_uri": {{
+                                "client_secret": {{
                                     "type": "string"
                                 }},
+                                "redirect_uri": {{
+                                    "type": "string",
+                                    "format": "uri"
+                                }},
                                 "irods_user_claim": {{
                                     "type": "string"
                                 }},
                                 "tls_certificates_directory": {{
                                     "type": "string"
+                                }},
+                                "user_attribute_mapping": {{
+                                    "description": "Bleh blah",
```

Comment:
not sure what this is or what it should be.

Reply:
Noting here that we discussed this and landed on this should isn't a high priority item.

Let's capture it in an issue and investigate whether it seems helpful to users. As of right now, it doesn't offer much benefit because the it gets lost in the validation error output produced by jsonschema.

Reply:
Captured in issue #262 .